src/main/kotlin/org/rust/ide/docs/RsDocumentationProvider.kt

```diff
@@ -241,7 +241,7 @@ private val RsItemElement.declarationModifiers: List<String> get() {
             if (isConst) {
                 modifiers += "const"
             }
-            if (isUnsafe) {
+            if (isActuallyUnsafe) {
```

Comment:
Oh, it's a bug that we used `isUnsafe` in the old meaning because we should add `unsafe` modifier only in a case when it is written in a function declaration. 
So, let's keep old code here and fix related tests